redirects.json

```diff
@@ -0,0 +1,277 @@
+    "destination": "sdk-onramp-kit/reference/OnRampKitBasePack",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-aa-sdk/onramp-kit/reference/moneriumpack",
+    "destination": "sdk-onramp-kit/reference/MoneriumPack",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-aa-sdk/relay-kit",
+    "destination": "/sdk-relay-kit",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-aa-sdk/relay-kit/guides/gelato",
+    "destination": "/sdk-relay-kit/guides/gelato-relay",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-aa-sdk/relay-kit/reference/v2",
+    "destination": "/sdk-relay-kit/reference/migrating-to-v2",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-aa-sdk/api-kit",
+    "destination": "/sdk-api-kit",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-aa-sdk/api-kit/reference",
+    "destination": "/sdk-api-kit/reference",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-aa-sdk/api-kit/reference/v1",
+    "destination": "/sdk-api-kit/reference/migrating-to-v1",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-aa-sdk/api-kit/reference/v2",
+    "destination": "/sdk-api-kit/reference/migrating-to-v2",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-protocol/safe-core-protocol",
+    "destination": "/protocol-overview",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-api/service-architecture",
+    "destination": "/api-service-architecture",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-api/service-architecture/safe-transaction-service",
+    "destination": "/api-service-architecture/safe-transaction-service",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-api/rpc-requirements",
+    "destination": "/api-rpc-requirements",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-api/supported-networks",
+    "destination": "/api-supported-networks",
+    "permanent": true
+  },
+  {
+    "source": "/safe-apps-sdk/safe-apps",
+    "destination": "/apps-sdk-overview",
+    "permanent": true
+  },
+  {
+    "source": "/safe-apps-sdk/get-started",
+    "destination": "/apps-sdk-get-started",
+    "permanent": true
+  },
+  {
+    "source": "/safe-apps-sdk/release",
+    "destination": "/apps-sdk-release",
+    "permanent": true
+  },
+  {
+    "source": "/miscellaneous/bug-bounty-program",
+    "destination": "/bug-bounty",
+    "permanent": true
+  },
+  {
+    "source": "/miscellaneous/bug-bounty-program/past-paid-bounties",
+    "destination": "/bug-bounty/past-paid-bounties",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-protocol",
+    "destination": "/sdk-overview",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-protocol/plugins",
+    "destination": "/smart-account-modules",
+    "permanent": true
+  },
+  {
+    "source": "/safe-core-protocol/hooks",
+    "destination": "/smart-account-guards",
+    "permanent": true
+  },
```

Comment:
I think these redirects can be removed:
- Pointing to the SDK overview from the old Protocol link doesn't make much sense for me.
- Redirecting from Protocol Plugins to Safe Modules and from Protocol Hooks to Safe Guards is not 100% correct and it would create more confusion for people following the link, as they are different things that are not replaceable with the new link.

I would say we remove these or redirect to the root of the new docs if we don't want to show the 404 Error.

Reply:
Ok, these redirections were taken from here: https://github.com/safe-global/safe-docs/blob/main/.gitbook.yaml

I don't really have an opinion on keeping/removing them except for continuity with old links that may remain somewhere on the internet. Any thoughts on this @tanay1337 ?

Reply:
Adding more context from my side:
I think we should take notes of the old URLs, remove them for now as they are pointing to the wrong content right now, and add the redirects again once we include the new content for the Protocul in H2